src/components/forms/PractitionerForm/UserIdSelect/index.tsx

```diff
@@ -80,28 +87,40 @@ export const UserIdSelect = (props: Props) => {
     }
   };
 
+  // fetch total number of users
+  const fetchUsersCount = async () => {
+    const serve = new props.serviceClass(OPENSRP_USERS_COUNT_ENDPOINT);
+    const response = await serve.list().catch(err => {
+      displayError(err);
+    });
+    return response;
+  };
+
   /** Pulls all users data */
-  const loadUsers = async (service: typeof OpenSRPService = OpenSRPService) => {
+  const loadUsers = async () => {
     let filterParams = {
       page_size: USERS_REQUEST_PAGE_SIZE,
+      source: OPENSRP_KEYCLOAK_PARAM,
       start_index: 0,
     };
-    const serve = new service(OPENSRP_USERS_ENDPOINT);
+    const serve = new props.serviceClass(OPENSRP_USERS_ENDPOINT);
     const allUsers = [];
-    let response: UserResponse;
+    let response: User[];
+    const usersCount: number = await fetchUsersCount();
```

Comment:
If `fetchUsersCount` encounters an error, will this break?

Reply:
Refactored this to allow handling of failed requests to `user/count` endpoint via https://github.com/onaio/reveal-frontend/pull/1166/commits/4f8ec26fd253c9f38235913d0f35e043364545c9